### ISSUE TITLE
Please support another schema in target PostgreSQL rather than public

### ISSUE THREAD

**Issue by abrilliantov** [slotix/dbconvert-streams-public#2]
I tried following json

```
{
 "source": {
   "mode": "convert",
   "type": "mysql",
   "connection": "root:pass@tcp(host.docker.internal:3306)/source_db"
 },
 "target": {
   "type": "postgresql",
   "connection": "postgres://postgres:postgres@host.docker.internal:5432/target_db/target_schema"
 }
}
```

but got error
```
target_db/target_schema does not exist
```

**Comment by slotix** [comment]
### Example DSN

`user=postgres password=postgres host=host.docker.internal port=5432 dbname=target_db`

### Example URL
`postgres://postgres:postgres@host.docker.internal:5432/target_db`

The connection can be in URL format or keyword = value format (DSN style). 

See more at https://stream.dbconvert.com/sources/postgresql/postgresql-server#postgresql-specific-options


It's not possible to specify the schema directly in the connection string, but you can specify the schema inside the filter:

```{
 "source": {
   "mode": "convert",
   "type": "mysql",
   "connection": "root:pass@tcp(host.docker.internal:3306)/source_db",
   "filter": {
      "tables": [
        { "name": "private.products1", "operations": ["insert", "update", "delete"]},
        { "name": "products2", "operations": ["insert", "update", "delete"]}
      ]
    }
 },
 "target": {
   "type": "postgresql",
   "connection": "postgres://postgres:postgres@host.docker.internal:5432/target_db"
 }
}
```

**Comment by PekopT** [comment]
@slotix so, basically, I need to specify all my 131 tables there in `filters` ?

That's definitely not appropriate, especially in fast-moving startup environments, where number of tables can grow +-10 each week

**Comment by slotix** [comment]
@PekopT You're absolutely right. Managing 131 tables individually in filters sounds like a recipe for chaos. We are developing a new version with a simpler approach to specifying PostgreSQL schemas.

**Comment by mstephano** [comment]
Like @PekopT said, I would also appreciate a version where we don't need to specify the 200 tables in the filters.

**Comment by slotix** [comment]
Closing this to tidy up the tracker after a long time open. If targeting a schema other than public is still blocking you, reply here or write to db-support@dbconvert.com and we will look at it again.
